### ISSUE TITLE
Diacritics to revisit

### ISSUE THREAD

**Issue by bambooforest** [phoible/dev#213]
Some combining diacritics don't seem to appear in Phoneme, but do in Allophones:

velarized/pharyngealized (combining tilde overlay)
`phoible %>% filter(grepl("̴", Phoneme)) %>% select(Phoneme) %>% distinct()`

Doesn't appear anywhere afaict:

denasalized (combining not tilde above)
`phoible %>% filter(grepl("͊", Phoneme)) %>% select(Phoneme) %>% distinct()`

nasal emission (combining homothetic)
`phoible %>% filter(grepl("͋", Phoneme)) %>% select(Phoneme) %>% distinct()`

derhoticized (combining breve below)
`phoible %>% filter(grepl("̮", Phoneme)) %>% select(Phoneme) %>% distinct()`

This appears in one phoneme type in one language (638 NENETS):

stiff (combining caron below)
`phoible %>% filter(grepl("̬", Phoneme)) %>% select(Phoneme) %>% distinct()`

This appears in two languages
phoible %>% filter(grepl("̚", Phoneme)) %>% select(InventoryID, LanguageName, Phoneme)

  InventoryID LanguageName Phoneme
1        1411       Ngomba       q̚
2        1567      Tangale       d̪̚

**Comment by drammock** [comment]
Yes, I noticed this too.  Things showing up in allophones but not phonemes is not all that surprising, so I didn't worry about it much. Here are some comments:

- I'd prefer to get rid of `combining tilde overlay` if we can, since we have dedicated, separate modifiers for "velarized" and "pharyngealized".  But knowing which one to pick requires going back to the sources.
- I already removed (by commenting out) `derhotacized` [here](https://github.com/phoible/dev/blob/master/scripts/aggregation-helper-functions.R#L396)) and `nasal emission` [here](https://github.com/phoible/dev/blob/master/scripts/aggregation-helper-functions.R#L409)... I must have missed `denasalized` in my testing.
- I also noticed the rarity of `stiff` and would like to dig deeper into it
